src/server/server_family.cc

```diff
@@ -2452,6 +2455,11 @@ string ServerFamily::FormatInfoMetrics(const Metrics& m, std::string_view sectio
     uint64_t uptime = time(NULL) - start_time_;
     append("uptime_in_seconds", uptime);
     append("uptime_in_days", uptime / (3600 * 24));
+
+    append("hz", GetFlag(FLAGS_hz));
+    append("executable", base::kProgramName);
+    absl::CommandLineFlag* flagfile_flag = absl::FindCommandLineFlag("flagfile");
+    append("config_file", flagfile_flag ? flagfile_flag->CurrentValue() : "");
```

Comment:
i believe this flag always exists
```suggestion
    append("config_file", flagfile_flag->CurrentValue());
```